### DISCUSSION TITLE
Which apk in the releases would i install for a fire hd 10?

### DISCUSSION THREAD

**Discussion by Nonethousand** [termux/termux-app#2531] [Q&A] [answered]
title

**Comment by ChiefMikeK** [comment, marked as answer]
You can download [Termux from F-Droid](https://f-droid.org/en/packages/com.termux/ )

**Comment by agnostic-apollo** [comment]
Depends on which year's model do you have and which android version it is on. 2017 or 2021?

**Reply by Nonethousand** [reply]
> Depends on which year's model do you have and which android version it is on. 2017 or 2021?

i don't know, my uncle gave it to me and i threw away the box

**Reply by agnostic-apollo** [reply]
Check android settings -> About

**Reply by Nonethousand** [reply]
> Check android settings -> About

It just says 11th generation. Idk if that will help

**Reply by agnostic-apollo** [reply]
11 generation should be the 2021 one with android 9 likely. Check about page for android version confirmation. For any android version >= 7, use F-Droid or Github `v0.118.0`.

https://github.com/termux/termux-app#f-droid
